src/ODBC/OdbcNativeMetadataProvider.php

```diff
@@ -117,8 +117,8 @@ protected function processColumnData(
         $type = $data['TYPE_NAME'];
         if (!in_array(strtolower($type), $this->typesWithoutLength)) {
             $length = $data['COLUMN_SIZE'] ?? null;
-            if ($length && isset($data['NUM_PREC_RADIX'])) {
-                $length .= ',' . $data['NUM_PREC_RADIX'];
+            if ($length && isset($data['DECIMAL_DIGITS'])) {
+                $length .= ',' . $data['DECIMAL_DIGITS'];
```

Comment:
MySQL oproti ktorej je to testovane v tomto repe rovnaku hodnotu (je to jedno), ale v `DB2` to nie je jedno.